modules/apps/transfer/module.go

```diff
@@ -92,7 +93,9 @@ func (AppModuleBasic) GetQueryCmd() *cobra.Command {
 // AppModule represents the AppModule for this module
 type AppModule struct {
 	AppModuleBasic
-	keeper keeper.Keeper
+	keeper       keeper.Keeper
+	scopedKeeper capabilitykeeper.ScopedKeeper
+	app          porttypes.IBCModule
```

Comment:
can you add comments for each of these fields and explain what they do in the context of the `AppModule`?